full-service/src/json_rpc/transaction_log.rs

```diff
@@ -113,7 +107,7 @@ impl TransactionLog {
 
 #[derive(Deserialize, Serialize, Default, Debug, Clone)]
 pub struct InputTxo {
-    pub txo_id_hex: String,
+    pub id: String,
```

Comment:
This is potentially a breaking API change. Can we move these changes to another PR and specify the migration plan?

Reply:
No direct db migration plan, as this is part of the 2.0 release. As far as the changes for txo_id_hex -> id, I can move that to another PR